index.html

```diff
@@ -65,54 +77,69 @@ <h4>Skills</h4>
     </section>
     <section id="recent-works">
         <h2>MY RECENT WORKS</h2>
-        <hr>        <div id="recent-works-grid">
+        <hr id="hr2">        
+        <div id="recent-works-grid">
             <div class="works" id="work-1">
-                <img src="Works/work1.png" alt="First work image">
-                <h4>Personal project</h4>
-                <p>This is a project with CSS. In this project, I created a 
-                    replica of 
-                    <a href="https://jolly-kalam-23776e.netlify.app/cssgridresponsive/#section-b">this webpage</a>.
-                    Feel free to checkout the link and compare it with the final work.
-                </p>
-                <ul class="project-languages">
-                    <li class="project-languages-1">html</li>
-                    <li class="project-languages-1">css</li>
-                </ul>
-                <div class="n1"><button>See Project</button></div>
+                <div class="desktop-layout">
+                    <img src="Works/work1.png" alt="First work image">
+                    <div class="desktop-layout-1">
+                        <h4>Personal project</h4>
+                        <p>This is a project with CSS. In this project, I created a 
+                            replica of 
+                            <a href="https://jolly-kalam-23776e.netlify.app/cssgridresponsive/#section-b">this webpage</a>.
+                            Feel free to checkout the link and compare it with the final work.
+                        </p>
+                        <ul class="project-languages">
+                            <li class="project-languages-1">html</li>
+                            <li class="project-languages-1">css</li>
+                        </ul>
```

Comment:
- [ ] [_Optional_] Kindly remove this extra line at the end of your project languages. It's not in accordance with the Figma templates

![image](https://user-images.githubusercontent.com/90556221/166121989-d12e5e7b-a511-40d4-9954-5ffec4631f34.png)